eProctoring/user/views.py

```
@@ -7,11 +7,12 @@
 from django.contrib.auth.decorators import login_required
 from django.contrib.sessions.models import Session
 
-
+@require_http_methods(["GET", "POST"])
```

Ruff (0.8.2)

Undefined name `require_http_methods`

(F821)